StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1501UnitTests.cs

```diff
@@ -1017,7 +1017,7 @@ protected override IEnumerable<string> GetDisabledDiagnostics()
         /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
         [Theory]
         [MemberData(nameof(TestStatements))]
-        private async Task TestNoSA1503CodeFixForStatementAsync(string statementText)
+        public async Task TestNoSA1503CodeFixForStatementAsync(string statementText)
```

Comment:
This currently triggers SA1202. This method should be placed above
 `protected override IEnumerable<DiagnosticAnalyzer> GetCSharpDiagnosticAnalyzers()`

Reply:
Fixed in e578a1a697b01981ec7bb22a97297bcbe1a79502